app/views/group_plan_summary.scala.html

```diff
@@ -14,74 +14,106 @@
+								ActionItem(
+									href = routes.GroupSchemeController.editCompany(index).url,
+									content = Text(messages("ers.change_this")),
+									visuallyHiddenText = Some(messages("ers.change_this"))
+								),
+								ActionItem(
+									href = routes.GroupSchemeController.deleteCompany(index).url,
+									content = Text(messages("ers.delete")),
+								)
+							)
+						))
+					)
+					val addressSummary = SummaryListRow(
+						key = Key(
+							content = Text(messages("ers_group_summary.address"))
+						),
+						value = Value(
+							content = Text(ersUtil.buildAddressSummary(company))
+						)
+					)
+					val crnSummary = SummaryListRow(
+						key = Key(
+							content = Text(messages("ers_group_summary.crn"))
+						),
+						value = Value(
+							content = Text(company.companyReg.getOrElse(""))
+						)
+					)
+					val ctrSummary = SummaryListRow(
+						key = Key(
+							content = Text(messages("ers_group_summary.ctr"))
+						),
+						value = Value(
+							content = Text(company.corporationRef.getOrElse(""))
+						)
+					)
 
-				<tr>
-				  <td class="group-plan-name-summary-text-align">@Html(messages("ers_group_summary.address"))</td>
-				  <td class="group-plan-name-summary-text-align">
-					  @company.addressLine1@if(company.addressLine2.getOrElse("") != ""){, @company.addressLine2}@if(company.addressLine3.getOrElse("") != ""){, @company.addressLine3}@if(company.addressLine4.getOrElse("") != ""){, @company.addressLine4}@if(company.postcode.getOrElse("") != ""){, @company.postcode}@if(company.country.getOrElse("") != ""){, @countryCodes.getCountry(company.country.get)}
-				  </td>
-				</tr>
-				@if(company.country.isDefined){@if(company.country.get == ersUtil.DEFAULT_COUNTRY) {
-					@if(company.companyReg.isDefined) {
-						<tr>
-							<td class="group-plan-name-summary-text-align">@Html(messages("ers_group_summary.crn"))</td>
-							<td class="group-plan-summary-text-align">@company.companyReg</td>
-						</tr>
-					}
-					@if(company.corporationRef.isDefined) {
-						<tr>
-							<td class="group-plan-name-summary-text-align">@Html(messages("ers_group_summary.ctr"))</td>
-							<td class="group-plan-summary-text-align">@company.corporationRef</td>
-						</tr>
-					}
-				}}
-			  </tbody>
-			</table>
-			}
-			@if(groupSchemeType == ersUtil.OPTION_CSV) {
-			<h2>@Html(messages("ers_group_summary.notice"))</h2>
-			<p>@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_SUMMARY, "notice_info"))&nbsp;<a href="routes.GroupSchemeController.groupCheckCSVFilePage()" id="check-another-return-file-link">@Html(messages("ers_group_summary.notice_info2"))</a>.<p/>
-			}
-			<div @if(groupSchemeType == ersUtil.OPTION_MANUAL) {class="group-plan-summary-margin"}>
-			<button role="button" id="continue" class="button" type="submit" @if(companies.companies.isEmpty) {disabled="disabled"}>@Html(messages("ers.continue"))</button>
-			</div>
+					Seq(companyNameSummary, addressSummary) ++
```

Comment:
comment as a reminder (not for this ticket): we'll need a ticket to stop sending the crn/ctr onwards if the user selected a country outside the UK, atm it's not displaying it but likely still sending it on